example/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -8,14 +8,16 @@ import {
   previewScreenLabels,
   soundOutputDevicesLabels,
 } from '../../../types/ComponentLabels';
+
 import {
-  AddPeerRequest,
-  Configuration,
   ConfigurationParameters,
+  Configuration,
+  RoomApiFp,
+  AddPeerRequest,
   PeerDetailsResponseData,
   Room,
-  RoomApiFp,
-} from '../../server-api';
+} from '@fishjam-dev/js-server-sdk';
```

Comment:
We should verify why our CI checks allowed this incorrect import order. I think it is defined as one of eslint/prettier rules 🤔

Reply:
I do not think it was defined anywhere.